### PR TITLE
📝 Update usage guide

### PR DESCRIPTION
**Pull request by lmachens**
The README.md should give all the information to contribute to this project. 
Examples are added.

`COURSE_URL` is not used anymore (might be used in the future).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,24 +4,49 @@
...
+
+Example:  
+`https://talents.neuefische.com/student/73df25e3-c6db-4267-b270-a480e38ac6dc`  
+The `studentId` is `73df25e3-c6db-4267-b270-a480e38ac6dc`.
+
+To select a course, you need to add `course` to the query string.
+
+`http://localhost:3030/?id={studentId}&course={courseId}`
+
+The `courseId` is the id of the course you want to select from `src/assets/samples/courses.json`.
```
Comment (badabam) [inline README.md:44]: I'd suggest to have the `course` in all examples of this document as the default (is it the first entry of the courses array?)is not likely to fit. We might even consider removing the default and show an error with a message that hints to the format with `id` and `course`.

```suggestion
To generate a certificate for a student, visit `http://localhost:3030/?id={studentId}&course={courseId}`.

| param | explanation | example |
|---|---|---|
| `id` | studentId that is used in the talent app | `73df25e3-abc1-1234-b270-a480e38ac6dc` |
| `course` | courseId is defined in `src/assets/samples/courses.json` | `hh-web-21-3` |

Example:  
`http://localhost:3030/?id={studentId}&course={courseId}`
```

**Review by badabam — COMMENTED** [review]
Nice work! I made a suggestion, feel free to follow or reject it.

**Review by badabam — APPROVED** [review]
I wanted to approve this – even though I made a suggestion.

File: README.md (hunk 1 above)
Comment (lmachens) [reply README.md]: Thank you, I added the table and did some small further adjustments.

**Pull request merged**